packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -1,4 +1,8 @@
 import { L1Network, L2Network, addCustomNetwork } from '@arbitrum/sdk'
+import {
+  l1Networks,
+  l2Networks
+} from '@arbitrum/sdk/dist/lib/dataEntities/networks'
```

Comment:
Always wondered, is this a good and reliable way to import something which is not exported directly from `arbitrum/sdk `main interface?
I had to create a separate map of chainId's I required in one of my PR's to avoid this. What do you all think?

Reply:
it's a bit ugly but other than that it's alright

we can check if it's possible to change the module resolution setting in the tsconfig of the sdk
if it doesn't have any negative effect, it might be good to change it

Reply:
Ideally, it should be exported from the SDK directly

Reply:
Think it's good enough for this PR, but creating a ticket to improve types on `getL1Network` and `getL2Network` which should solve this issue for us!